packages/now-static-build/test/fixtures/10-gatsby-without-build-script/now.json

```diff
@@ -0,0 +1,14 @@
+{
+  "version": 2,
+  "builds": [
+    {
+      "src": "package.json",
+      "use": "@now/static-build",
+      "config": {
+        "zeroConfig": true,
+        "framework": "gatsby"
```

Comment:
So the idea is that the `framework` comes from the Project?

Reply:
@styfle correct, or whatever the user provided